rkt/image_gc.go

```diff
@@ -21,6 +21,7 @@ import (
 
 	"github.com/coreos/rkt/common"
 	"github.com/coreos/rkt/pkg/lock"
+	pkgPod "github.com/coreos/rkt/pkg/pod"
```

Comment:
@s-urbaniak My argument for using aliases is to make it very distinguishable from the variable name `pod` that appears in a lot of places.